en/docs/api-analytics/samples/publishing-analytics-events-to-external-systems.md

```diff
@@ -1,250 +1,276 @@
----
-title: Publishing Analytics Events to External Systems - API Manager Documentation 4.0.0
----
-
 # Publishing Analytics Events to External Systems
 
-## Introduction
-
-Instead of publishing analytics events to the cloud, It is also possible to log the same events and publish them to external systems. This guide will explain the steps required to do it. For demonstration purposes, we have selected ELK as the external system.
-
-This section will cover the steps required to create a sample, configure the created sample with WSO2 API-M or Choreo Connect, and then publish it to an external System (ELK).
-
-## Creating the Sample
-
-You have to create a new `Java/Maven project`. 
+WSO2 API Manager allows publishing its analytics data to external systems in the same way it publishes the data to the cloud. For this purpose you need to create a custom event publisher. This guide will explain and walk through the steps required to implement, deploy, and configure a custom event publisher.
 
-With the use of `WSO2 Analytics Publisher` extension which is available at `WSO2 nexus` repository it is possible to create the sample. Created sample can be used to log and publish analytics events to external systems.
+For demonstration purposes, let’s assume ELK as the external system.
 
-There is an already [created sample](https://github.com/wso2/samples-apim/tree/master/analytics-event-publisher) and if you wish to use that sample instead of developing the sample from scratch, then you can ignore the steps of creating the sample and start from [here]({{base_path}}/api-analytics/samples/publishing-analytics-events-to-external-systems/#build-the-project).
+## Step 1 - Create an Event Publisher
```

Comment:
@lasanthaS , I added numbering to the doc to bring in a flow to the content. Please check and confirm the numbering.

Reply:
Numbering is fine. Let's proceed